### DISCUSSION TITLE
Can fzf-tab show all results when there're only a few matches?

### DISCUSSION THREAD

**Discussion by nabn** [Aloxaf/fzf-tab#349] [Q&A]
When I'm in a directory with too few directories (<4) and try cd <TAB>, the options list is one option tall. Rest are still accessible when tabbed.

In this screen recording, I'm progressively adding directories and checking what the completion prompt gives me:

https://user-images.githubusercontent.com/1693287/210297440-8a7062d8-ac56-4856-862e-fa0cbaf33c2c.mp4

I'd expect that when there's enough room for all options, all of them are displayed on the screen. Was that the intention?

I found [an issue](https://github.com/Aloxaf/fzf-tab/issues/147) that was closed.

**Comment by Aloxaf** [comment]
Just increase bottom [popup-pad](https://github.com/Aloxaf/fzf-tab/wiki/Configuration#popup-pad)

example:
```
zstyle ':fzf-tab:complete:cd:*' popup-pad 30 2
```
